src/panels/lovelace/editor/config-elements/hui-conditional-card-editor.ts

```diff
@@ -185,8 +189,8 @@ export class HuiConditionalCardEditor
                           : cond.state}
                         .idx=${idx}
                         .configValue=${"state"}
-                        @value-changed=${this._changeCondition}
-                      ></paper-input>
+                        @input=${this._changeCondition}
```

Comment:
Why not change event?

Reply:
Because we want to show a preview right away?